docs/book/starter-guide/pipelines/parameters-and-caching.md

```diff
@@ -4,24 +4,25 @@ description: Iteration is native to ZenML.
 
 # Quickly iterating in ZenML
 
-Machine learning pipelines are rerun many times over throughout their development lifecycle. 
+Machine learning pipelines are rerun many times over throughout their development lifecycle.
 
 ## Parameterizing steps
 
-In order to iterate quickly, one must be able to quickly tweak pipeline runs by changing various 
-parameters for the steps that make up your pipeline.
+In order to iterate quickly, one must be able to quickly tweak pipeline runs by
```

Comment:
Somewhere in this section, you could show an example of a step that could benefit from parameterization, then show the parameterized version. For instance, you could have a step that uses some value hard-coded in the step implementation, that could be extracted as a parameter. 

I would also use the example to justify why parameterization is a good thing to have or generally why separation of code from configuration is a good practice.